### PR TITLE
Change publisher list interval from 1 to 3 days

### PR DESCRIPTION
**Pull request by NejcZdovc**
Resolves https://github.com/brave/brave-browser/issues/8955

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: components/brave_rewards/browser/static_values.h [hunk 1]
```diff
@@ -34,7 +34,7 @@ const std::vector<std::string> kOnlyAnonWalletCountries = {
 
   const std::map<std::string, uint64_t> kUInt64Options = {
       {ledger::kOptionPublisherListRefreshInterval,
-       base::Time::kHoursPerDay * base::Time::kSecondsPerHour}
+       7* base::Time::kHoursPerDay * base::Time::kSecondsPerHour}
```
Comment (NejcZdovc) [inline components/brave_rewards/browser/static_values.h:37]: mobile should be on 7 days

File: vendor/bat-native-ledger/src/bat/ledger/internal/static_values.h [hunk 2]
```diff
@@ -141,9 +141,6 @@ static const uint64_t _milliseconds_minute = 60 * 1000;
 
 static const uint64_t _milliseconds_second = 1000;
 
-// 24 hours in seconds
-static const uint64_t _publishers_list_load_interval = 24 * 60 * 60;
```
Comment (NejcZdovc) [inline vendor/bat-native-ledger/src/bat/ledger/internal/static_values.h:145]: removing as we don't use it anymore

**Comment by NejcZdovc** [comment]
CI failed on un related things

**Comment by kjozwiak** [comment]
Verification **PASSED** on `macOS 10.15.3 x64 Catalina` using the following build:

```
Brave | 1.8.61 Chromium: 81.0.4044.83 (Official Build) nightly (64-bit)
-- | --
Revision | 5718ca617723806b9aacfb41108a0d911d1bde74-refs/branch-heads/4044@{#853}
OS | macOS Version 10.15.3 (Build 19D76)
```

* ensured that the `publisher list` was downloaded on first launch and placed into `server_publisher_info` via `publisher_info_db`
* changed `server_publisher_list_stamp` via `Preferences` to `1585724400`
* restarted Brave and ensured that `Publisher list` wasn't being downloaded
* changed `server_publisher_list_stamp` via `Preferences` to `1585378800`

Example of `Publisher List` being downloaded after moving `server_publisher_list_stamp` three days back:

```
[ RESPONSE - OnDownload ]
> time: 1585807800
> result: Success
> http code: 200
> response: Publisher list
> headers accept-ranges: bytes
> headers access-control-allow-headers: Origin, X-Requested-With, Content-Type, Accept, Authorization
> headers access-control-allow-methods: GET
> headers access-control-allow-origin: https://rewards.bravesoftware.com
> headers access-control-request-method: GET
> headers age: 3392
> headers cache-control: max-age=3600, public
> headers content-encoding: gzip
> headers content-length: 16931839
> headers content-type: application/json; charset=utf-8
> headers date: Thu, 02 Apr 2020 06:09:58 GMT
> headers etag: W/"f0eca12d54cdfe349c6e71380306fe54"
> headers server: Cowboy
> headers status: 200
> headers strict-transport-security: max-age=31536000; includeSubDomains
> headers vary: Accept-Encoding
> headers via: 1.1 varnish
> headers x-cache: HIT
> headers x-cache-hits: 0
> headers x-frame-options: deny
> headers x-request-id: fb7d5271-2a70-4997-a357-52a5c14a6a81
> headers x-runtime: 2.010515
> headers x-served-by: cache-yyz4547-YYZ
[ END RESPONSE ]
```

**Pull request merged**